Client/src/Components/DataCell/DataCell.js

```diff
@@ -49,11 +49,10 @@ export default function DataCell() {
             await fetch('http://localhost:4000/api/v1/dataExtract', options)
                 .then(resp => resp.json())
                 .then(result => {

```

Comment:
This file contains a rejection if a file isnt json. We dont need that rejection since we will support 5 file types

png, txt, csv, json, pdf

Reply:
@TiraFire 's PR includes removal of this file in favor of DataCell.tsx, and then after that this page should only accept .png or array of pngs